code/modules/chemistry/tools/iv_drips.dm

```diff
@@ -67,8 +67,8 @@
 			src.UpdateOverlays(null, "inj_dr")
 		signal_event("icon_updated")
 
-	is_open_container()
-		return 1
+	is_open_container(inward)
+		return TRUE
```

Comment:
these should be replaced with the inward flag

Reply:
i've changed its function slightly, since the original code was just it always returning true. Now it checks `src.slashed`. Stuff can go in when poured in, but can only pour out if slashed.